tools/remote_build/rbe_common.bazelrc

```diff
@@ -81,6 +81,10 @@ build:tsan --extra_execution_platforms=//third_party/toolchains:rbe_ubuntu1604,/
 # undefined behavior sanitizer: most settings are already in %workspace%/.bazelrc
 # we only need a few additional ones that are Foundry specific
 build:ubsan --copt=-gmlt
+# workaround to use libc++ instead of libstdc++ because gcc 4.9 installed in ubuntu16_04_clang
```

Comment:
file an issue against https://github.com/bazelbuild/bazel-toolchains because that's where @bazel_toolchains//configs/experimental/ubuntu16_04_clang is hosted?

Reply:
Also, you're saying gcc4.9 but we're using clang to build?https://github.com/grpc/grpc/blob/c379323b3140cb62efe7c1aa9ce984fc575cc7b3/tools/bazel.rc#L6

Reply:
Ok, let me know once this is ready for review.